components/dashboard/src/components/podkit/buttons/Button.tsx

```diff
@@ -10,7 +10,7 @@ import { cva, VariantProps } from "class-variance-authority";
 import { cn } from "@podkit/lib/cn";
 
 export const buttonVariants = cva(
-    "inline-flex items-center justify-center whitespace-nowrap rounded-lg text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-4 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50",
+    "inline-flex gap-1.5 items-center justify-center whitespace-nowrap rounded-lg text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-4 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50",
```

Comment:
Might be better to handle this in the specific place we have this button so we don't cause all buttons to inherit this unexpectedly, i.e. putting a span around the button contents w/ flex styles and the gap. Alternatively we might want to audit all of our buttons to see how this effects them.

Reply:
Thanks to you both for noticing the obvious problem with this. Will move to the button instance instead to avoid oopsies.